docs/guide/custom_map_rotate.rst

```diff
@@ -1,8 +1,8 @@
-.. _map_rotate_custom:
+.. _how-to-add-a-new-rotation-method-to-map:
 
-****************************
-Adding a new rotation method
-****************************
+***************************************
+How to add a new rotation method to Map
```

Comment:
When I started adding entries to the how-to section, I structured the titles such that they didn't include "how to" but that adding "how to" to the front of them made grammatical sense, e.g. "search the VSO" is the title and answers the question "how to search the VSO". I did this because the "how to" is implicit once you're in the how-to section and it also avoids repetition in the how-to toc tree. 

I suggest also doing that here and we can debate a reformatting of the titles later.

Reply:
Oh wait I'm getting confused. These are now in "Topic Guides" not how-tos so they should definitely not have "how-to" in the name.

Reply:
Yeah I was confused the first move, updated everything to be wrong. Hopefully now it's all fixed.